sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -52,71 +52,256 @@ export interface HttpTextFormat {
     inject(spanContext: SpanContext, format: string, carrier: unknown): void;
 }
 
+// @public (undocumented)
```

Comment:
I'm guessing you also saw this. Can you add documentation to new things you added? I think we'll need an issue to make sure we go back and add documentation to all the public APIs before GA.